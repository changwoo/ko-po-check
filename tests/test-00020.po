# -*-text-*-
# 괄호
#
msgid ""
msgstr ""
"Project-Id-Version: test 13\n"
"POT-Creation-Date: 2012-05-29 22:19+0900\n"
"PO-Revision-Date: 2012-05-29 22:19+0900\n"
"Last-Translator: Changwoo Ryu <cwryu@debian.org>\n"
"Language-Team: KO-PO-CHECK <cwryu@debian.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

msgid "Western style (parentheses)"
msgstr "맞춤법에서는 (이렇게) 띄어쓰지 않는다"

msgid "Western style (parentheses) again"
msgstr "이렇게(붙여 써야) 한다"

msgid "In western context, it is ok (ok)"
msgstr "In western context, it is ok (ok)"
