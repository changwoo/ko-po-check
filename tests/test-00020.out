test-00020.po:16: 맞춤법 규정에 따라 괄호 앞에 띄어 쓰지 않습니다
